api/SVGTRefElement.json

```diff
@@ -44,7 +44,7 @@
             "version_removed": "2.0"
           },
           "webview_android": {
-            "version_added": "1",
+            "version_added": "≤37",
```

Comment:
Due to https://github.com/mdn/yari/issues/3238 this is going to end up looking broken on MDN. This was in Safari 1, so I'm guessing it was enabled with the rest of SVG in some early version of Android WebView? Can you identify that version to avoid this looking broken?

Reply:
Unfortunately I don't have access to many old Android versions at the moment, only 1.0, 1.5, and 4.4, so I can't determine a specific version yet...